src/coreclr/vm/comutilnative.cpp

```diff
@@ -674,6 +674,21 @@ UINT64   GCInterface::m_remPressure[MEM_PRESSURE_COUNT] = {0, 0, 0, 0};   // his
 // (m_iteration % MEM_PRESSURE_COUNT) is used as an index into m_addPressure and m_remPressure
 UINT     GCInterface::m_iteration = 0;
 
+FCIMPL0(INT64, GCInterface::_GetTotalPauseDuration)
+{
+    FCALL_CONTRACT;
+
+    FC_GC_POLL_NOT_NEEDED();
+
+    // Unsigned to signed conversion happened here
+
+    // GetTotalPauseDuration returned microseconds, but the TimeSpan constructor requires
+    // 100 nanosecond unit.
+    
+    return (int64_t)(GCHeapUtilities::GetGCHeap()->GetTotalPauseDuration() * 10);
```

Comment:
why are we doing this conversion here instead of in GC like the pause durations returned with the GetMemoryInfo API?